cms/templates/course_outline.html

```diff
@@ -48,8 +48,7 @@ <h2 class="title title-3">${_("This course was created as a re-run. Some manual
         </li>
       </ul>
     </div>
-  </div>
-  %endif
+  </div>  %endif
```

Comment:
CC: @caijim 

I didn't notice it at the time, but this turns out to be a syntax error;

`SyntaxException: Unterminated control keyword: 'if' in file '/edx/app/edxapp/edx-platform/cms/templates/course_outline.html' at line: 31 char: 1`